spec/features/clearance/user_signs_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+require "support/features/clearance_helpers"
+
+feature "User signs in" do
+  scenario "with valid email and password" do
+    create_user "user@example.com", "user", "password"
+    sign_in_with "user@example.com", "password"
+
+    expect_user_to_be_signed_in
+  end
+end
+
+private
```

Comment:
This doesn't really do anything -- there's no way to call this method anyway, so marking it as `private` is needless extra security.